src/components/app/list-new/AppListType.ts

```diff
@@ -0,0 +1,47 @@
+import { ResponseType } from '@devtron-labs/devtron-fe-common-lib'
+
+export interface ArgoAppListResult {
+    appName: string
+    clusterName: string
+    namespace: string
+    appStatus: string
+    syncStatus?: string
+    clusterId?: string
+}
+export interface ArgoAppListResponse extends ArgoAppListResult {
+    result?: ArgoAppListResult
+}
+
+export interface AppListResponse extends ResponseType {
+    result?: AppsListResult
+}
+
+interface AppsListResult {
+    clusterIds: number[]
+    applicationType: string //DEVTRON-CHART-STORE, DEVTRON-APP ,HELM-APP
```

Comment:
we should add the specific type rather than string

Reply:
It is a backend response should not be enumerate